srtcore/core.cpp

```diff
@@ -9713,10 +9703,244 @@ int srt::CUDT::processData(CUnit* in_unit)
             return -1;
     }
 
+    return 0;
+}
+
+CUDT::time_point srt::CUDT::getPacketPTS(void*, const CPacket& packet)
+{
+    return m_pRcvBuffer->getPktTsbPdTime(packet.getMsgTimeStamp());
+}
+
+SRT_ATR_UNUSED static const char *const rexmitstat[] = {"ORIGINAL", "REXMITTED", "RXS-UNKNOWN"};
```

Comment:
The `rexmitstat` is still used only in one function. Is there a reason for making it a global constant?

Reply:
It will be. The goal is to minimize changes that are about to come, especially if they involve moving lines to other places.

Reply:
Then needs to have an `s_` prefix.
A suggestion:
```suggestion
SRT_ATR_UNUSED static const char *const s_rexmitstat_str[] = {"ORIGINAL", "REXMITTED", "RXS-UNKNOWN"};
```

Reply:
Very good, I take it.